assets/src/components/shortcuts.js

```diff
@@ -10,6 +10,7 @@ const Shortcuts = ( { insertBlock } ) => {
 	const blocks = [
 		'amp/amp-story-text',
 		'core/image',
+		'amp/amp-story-title',
```

Comment:
Is this right to include as a shortcut? It would only be used on the first page, and it should be part of the post type template so that it is inserted automatically when you create a new story.